src/pages/Transfer/TransferOverview/TransferOverview.styles.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from 'styled-components';
+
+import { Card, Flex, theme } from '@/component-library';
+
+const StyledWrapper = styled(Flex)`
+  max-width: 560px;
```

Comment:
Not sure if this is the max-witdth we want (is different from swap form) @tomjeatt

Reply:
better to match the swap form

Reply:
I cannot change this because the tab for XCM is very big

Reply:
No worries then—it's a small thing, so fine to leave it as it is.